### ISSUE TITLE
bitnami-common 1.17.0 breaks all image tags

### ISSUE THREAD

**Issue by pyromaniac3010** [bitnami/charts#11828]
### Name and Version

bitnami/common 1.17.0

### What steps will reproduce the bug?

all charts which have:
```dependencies:
  - name: common
    repository: https://charts.bitnami.com/bitnami
    tags:
      - bitnami-common
    version: 1.x.x
```
are reporting image tag version "nil" back and break.
this was introduced with https://github.com/bitnami/charts/pull/11797
This also leads to big problems with a huge amount of other bitnami charts depending on common.
bitnami/redis is affected by this, too!

### What do you see instead?

no change for the image tag

### Additional information

https://github.com/bitnami/charts/pull/11797#issuecomment-1219193803
https://github.com/bitnami/charts/pull/11797

**Comment by pyromaniac3010** [comment]
@carrodher please revert the commit and release a 1.17.1 asap as all users with ci/cd pipelines like argo-cd aso have for example stopped redis instances at the moment and cannot really work around that. many charts use the recommended dependency with "version: 1.x.x"!

**Comment by Panplumousse** [comment]
Hello,
same issu here with bitnami/kube-prometheus

**Comment by pyromaniac3010** [comment]
how long does it take to update https://charts.bitnami.com/bitnami/index.yaml ?

**Comment by carrodher** [comment]
Sorry for the inconveneces. Initial changes from #11797 (`1.17.0`) were reverted at #11829 (`1.17.1`). The same changes were implemented again at #11830 but bumping the chart version in a major (`2.0.0`).

The version with the fix is already published, see
```console
$ helm repo update bitnami
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "bitnami" chart repository
Update Complete. ⎈Happy Helming!⎈

$ helm search repo bitnami/common
NAME          	CHART VERSION	APP VERSION	DESCRIPTION
bitnami/common	1.17.1       	1.17.1     	A Library Helm Chart for grouping common logic ...
```

And the new major will be available once reviewed and merged the PR
